lib/server.coffee

```diff
@@ -126,7 +124,7 @@ WebApp.connectHandlers.use (req, res, next) ->
 
 		Spiderable.originalRequest = req
 		url 		= Spiderable._urlForPhantom Meteor.absoluteUrl(), req.url
-		hash 		= SHA256 url
+		hash 		= hashCode url
```

Comment:
I thing you are right and vote for base64 (but I'm not sure if it is lighter than currently suggested hash algorithm)